docs/api/security/psa_attestation.md

```diff
@@ -1,6 +1,6 @@
 ## PSA initial attestation
 
-The PSA initial attestation service enables an application to prove a device's identity to a caller during the authentication process.
+The PSA initial attestation service enables an application to prove a device's identity to any server or device before communicating data, as part of the authentication process.
```

Comment:
my proposal:
The PSA initial attestation service enables an application to prove a device's identity to any server and application, as part of the authentication process.